src/help/guides/extension-development/model-reference.md

```diff
@@ -0,0 +1,269 @@
+| `users` | Number | The number of users subscribed to the manga. May be the same value as the `follows` field. |
+| `views` | Number | The number of views this manga has had. |
+| `hentai` | Boolean | Whether or not this manga is a hentai. If it is set to true, the manga will not be visible to the user unless the user enables mature manga. |
+| `relatedIds` | String Array | The IDs of related manga. |
+| `lastUpdate` | String | A string showing when the manga was last updated. It might either be an absolute date (`March 1, 2021`) or a relative date (`2 days ago`). |
+
+## MangaStatus
+
+An integer enum representing the various statuses a manga can have.
+
+### Values
+
+| Name | Value | Description |
+|------|-------|-------------|
+| `Completed` | 0 | A manga that has officially completed production. Does not mean that the current chapter list contains all chapters that the manga will ever have, as sources can add special pages posted on social media or scanlations for chapters in languages other than the official release language. |
+| `Ongoing` | 1 | A manga that is still releasing chapters. |
+
+## MangaTile
+
+A tile representing a manga. This is a more barebones version of [the Manga object](#manga), intended for things like
+homepages and searches which will not return most manga metadata.
+
+### Required Fields
+
+| Name | Type | Description |
+|------|------|-------------|
+| `id` | String | The given identifier of this Manga. This may be unique to the source which uses it. For example, one source may use the value '1234' to identify a manga, whereas another one may use the value 'One-Piece' to identify |
+| `image` | String | A URL pointing to a thumbnail which can be displayed to present the manga. |
+| `title` | [IconText](#icontext) | An Icon Text object representing the title of the manga. |
+
+### Optional Fields
+
+| Name | Type | Description |
+|------|------|-------------|
+| `subtitleText` | [IconText](#icontext) | An Icon Text object representing the subtitle of the manga. |
+| `primaryText` | [IconText](#icontext) | An Icon Text which can be shown as primary text to the thumbnail. This is rendered in the bottom left of the manga object on the view. |
+| `secondaryText` | [IconText](#icontext) | An Icon Text which can be shown as secondary text to the thumbnail. This is rendered on the bottom right of the manga object on the view. |
+
+## PagedResults
+
+An object representing a "page" of requests. There is no limit to how many mangas fit in a page, that is up to the
+source to decide. These objects use the [metadata](metadata.md) field.
+
+### Required Fields
+
+| Name | Type | Description |
+|------|------|-------------|
+| results | [MangaTile](#mangatile) Array | The mangas that were found in the page. |
+| metadata | any | The metadata to carry onto future calls of the function returning the object. Reference the [page on metadata](metadata.md) to learn more. |
+
+## SearchRequest
```

Comment:
Emphasize that `title` is the only field that is supported at the moment.
With the current development direction, likely everything else will be removed from this method. An Advanced Search method will have different fields.

Reply:
I thought it was mentioned that Paperback only supports the title.

Reply:
I think this is fair for now because it's only a week or so until the stateful update comes in and this needs to be rewritten anyways.